homeassistant/components/monarch_money/sensor.py

```diff
@@ -152,6 +154,14 @@ async def async_setup_entry(
 
     async_add_entities(entity_list)
 
+    platform = entity_platform.async_get_current_platform()
+    platform.async_register_entity_service(
+        GET_HOLDINGS_SERVICE_NAME,
+        schema={},
+        func="async_get_holdings",
+        supports_response=SupportsResponse.ONLY,
+    )
+
```

Comment:
From previous PR:

![image](https://github.com/user-attachments/assets/9dcf6446-d502-428c-a5aa-111342aca34b)

Reply:
So you say we pick devices, does that mean we pick HA devices? Shouldn't this be an action call on integration level, so you can pick the device, rather than action on entity level